Assets/GothicVR/Scripts/Phoenix/Interface/Vm/VmGothicExternals.cs

```diff
@@ -0,0 +1,456 @@
+        [MonoPInvokeCallback(typeof(PxVm.PxVmExternalCallback))]
+        public static void IntToString(IntPtr vmPtr)
+        {
+            var val = PxVm.pxVmStackPopInt(vmPtr);
+            PxVm.pxVmStackPushString(vmPtr, val.ToString());
+        }
+
+        [MonoPInvokeCallback(typeof(PxVm.PxVmExternalCallback))]
+        public static void FloatToString(IntPtr vmPtr)
+        {
+            var val = PxVm.pxVmStackPopFloat(vmPtr);
+            PxVm.pxVmStackPushString(vmPtr, val.ToString(CultureInfo.InvariantCulture));
+        }
+
+        [MonoPInvokeCallback(typeof(PxVm.PxVmExternalCallback))]
+        public static void FloatToInt(IntPtr vmPtr)
+        {
+            var val = PxVm.pxVmStackPopFloat(vmPtr);
+            PxVm.pxVmStackPushInt(vmPtr, (int)val);
+        }
+            
+        [MonoPInvokeCallback(typeof(PxVm.PxVmExternalCallback))]
+        public static void IntToFloat(IntPtr vmPtr)
+        {
+            var val = PxVm.pxVmStackPopInt(vmPtr);
+            PxVm.pxVmStackPushFloat(vmPtr, val);
+        }
+
+        [MonoPInvokeCallback(typeof(PxVm.PxVmExternalCallback))]
+        public static void PrintDebug(IntPtr vmPtr)
+        {
+            if (!FeatureFlags.I.ShowZspyLogs)
+                return;
+            
+            var message = PxVm.VmStackPopString(vmPtr);
+            Debug.Log($"[zspy]: {message}");
+        }
+        
+        [MonoPInvokeCallback(typeof(PxVm.PxVmExternalCallback))]
+        public static void PrintDebugCh(IntPtr vmPtr)
+        {
+            if (!FeatureFlags.I.ShowZspyLogs)
+                return;
+            
+            var message = PxVm.VmStackPopString(vmPtr);
+            var channel = PxVm.pxVmStackPopInt(vmPtr);
+            
+            Debug.Log($"[zspy,{channel}]: {message}");
+        }
+        
+        [MonoPInvokeCallback(typeof(PxVm.PxVmExternalCallback))]
+        public static void PrintDebugInst(IntPtr vmPtr)
+        {
+            if (!FeatureFlags.I.ShowZspyLogs)
+                return;
+
+            var message = PxVm.VmStackPopString(vmPtr);
+            Debug.Log($"[zspy]: {message}");
+        }
+        
+        [MonoPInvokeCallback(typeof(PxVm.PxVmExternalCallback))]
+        public static void PrintDebugInstCh(IntPtr vmPtr)
+        {
+            if (!FeatureFlags.I.ShowZspyLogs)
+                return;
+            
+            var message = PxVm.VmStackPopString(vmPtr);
+            var channel = PxVm.pxVmStackPopInt(vmPtr);
+            
+            Debug.Log($"[zspy,{channel}]: {message}");
+        }
+        
+#endregion
+
+
+        [MonoPInvokeCallback(typeof(PxVm.PxVmExternalCallback))]
+        public static void Wld_InsertNpc(IntPtr vmPtr)
+        {
+            var spawnpoint = PxVm.VmStackPopString(vmPtr);
+            var npcInstance = PxVm.pxVmStackPopInt(vmPtr);
+            
+            NpcCreator.I.ExtWldInsertNpc(npcInstance, spawnpoint);
+        }
+
+        public static void Wld_IsFPAvailable(IntPtr vmPtr)
```

Comment:
This function is missing the callback

```suggestion
        [MonoPInvokeCallback(typeof(PxVm.PxVmExternalCallback))]
        public static void Wld_IsFPAvailable(IntPtr vmPtr)
```